cpg-core/src/main/java/de/fraunhofer/aisec/cpg/TranslationResult.java

```diff
@@ -135,4 +132,9 @@ public void printBenchmark() {
   public TranslationConfiguration getConfig() {
     return translationManager.getConfig();
   }
+
+  @NotNull
+  public BenchmarkResults getBenchmarkResult() {
```

Comment:
Inconstistency in naming here. Result or Results?

Reply:
Results I would say